flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -34,24 +35,24 @@
 public interface Executor {
 
     /** Starts the executor and ensures that its is ready for commands to be executed. */
-    void start() throws SqlExecutionException;
+    void start() throws SqlClientException;
```

Comment:
nit: I think we can still use SqlExecutionException.

we can reduce the modifcation... and it only throws SqlExecutionException here?